pc-application/src/test/java/pl/agh/product/catalog/application/controller/book/create/AddBookControllerTest.java

```diff
@@ -48,6 +47,7 @@
     private static final MediaType APPLICATION_JSON_UTF8 = new MediaType(MediaType.APPLICATION_JSON.getType(),
             MediaType.APPLICATION_JSON.getSubtype(), Charset.forName("utf8"));
 
+    @WithMockUser(username = "john", roles = {"ADMIN"})
```

Comment:
Teraz niech już tak zostanie, ale na przyszłość: jeśli chcesz dodać mock user dla każdego testu w danej klasie, to wystarczy że dodasz adnotację raz na poziomie klasy - będzie ona automatycznie działała dla każdego testu.

Dodatkowo `username` w tym wypadku nas nie obchodzi, więc możemy użyć domyślnej wartości (czyli `user`) i zapisać naszą adnotację tak:

```suggestion
    @WithMockUser(roles = "ADMIN")
```